form/create_custom_choice_type.rst

```diff
@@ -0,0 +1,273 @@
+.. index::
+   single: Form; Custom choice type
+
+How to Create a Custom Choice Field Type
+========================================
+
+Symfony :doc:`ChoiceType </reference/forms/types/choice>` is a very useful type
+that deals with a list of selected options.
+The Form component already provides many different choice types, like the
+intl types (:doc:`LanguageType </reference/forms/types/language>`, ...) and the
+:doc:`EntityType </reference/forms/types/entity>` which loads the choices from
+a set of Doctrine entities.
+
+It's also common to want to re-use the same list of choices for different fields.
+Creating a custom "choice" field is a great solution - something like::
+
+    use App\Form\Type\CategoryChoiceType;
+
+    // ... from any type
+    $builder
+        ->add('category', CategoryChoiceType::class, [
+            // ... some inherited or custom options for that type
+        ])
+        // ... add more fields
```

Comment:
I don't think each collapse comment (``...``) needs extra information. I would remove the words in this one (and maybe on line 19 as well). In the rest of the docs, we're only adding the comment for places where the user must do something (as an emphasize), so this can confuse people.

Reply:
I've removed as I agree it's not useful, but kept the one line 19 because this is the only context hint in that example code.